README.md

```diff
@@ -1,25 +1,35 @@
 prometheus-kafka-consumer-group-exporter
 ========================================
 
-A [prometheus](https://prometheus.io/) exporter for Kafka's consumer group information.
+A [Prometheus](https://prometheus.io/) exporter for Kafka's consumer group
+information.
 
-Capability
-==========
-- Exports Kafka's consumer group information which can be obtained by executing `kafka-consumer-groups.sh`
-- Supports only new consumer(`--new-consumer` switch enabled by default) which uses Kafka broker as the offset checkpoint store
+Capabilities
+============
+ - Exports Kafka's consumer group information which can be obtained by
+   executing `kafka-consumer-groups.sh`
+ - Supports only new consumer(`--new-consumer` switch enabled by default) which
+   uses Kafka broker as the offset checkpoint store
 
 Export metrics
 ==============
-- `kafka_broker_consumer_group_current_offset` : Consuming offset of each consumer group/client/topic/partition based on committed offset
-- `kafka_broker_consumer_group_offset_lag` : Offset lag between the last log end offset and consuming point of each consumer group/client/topic/partition
+ - `kafka_broker_consumer_group_current_offset` : Consuming offset of each
+   consumer group/client/topic/partition based on committed offset
+ - `kafka_broker_consumer_group_offset_lag` : Offset lag between the last log
+   end offset and consuming point of each consumer group/client/topic/partition
 
 Supported Kafka versions
 ========================
-This exporter relies on `kafka-consumer-groups.sh` script that is shipped as part of Apache Kafka distribution.
-Here is the list of Apache Kafka versions which has been tested to use from this exporter.
+This exporter relies on `kafka-consumer-groups.sh` script that is shipped as
+part of Apache Kafka distribution.  Here is the list of Apache Kafka versions
+which has been tested to use from this exporter:
 
-- `0.10.0.1`
-- `0.9.0.1`
+ - `0.10.0.1`
+ - `0.9.0.1`
+
+Feel free to add a test in
+[`collector_test.go`](https://github.com/kawamuray/prometheus-kafka-consumer-group-exporter/blob/master/exporter/collector_test.go)
```

Comment:
the intention of adding this statement isn't clear to me.
in the first place we have to modify the core code to support the  new format right? :D